modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -618,10 +618,10 @@ public static int StepDecimals(real_t step)
         /// This can also be used to round a floating point
         /// number to an arbitrary number of decimals.
         /// </summary>
-        /// <param name="s">The value to stepify.</param>
+        /// <param name="s">The value to snap.</param>
         /// <param name="step">The step size to snap to.</param>
         /// <returns></returns>
-        public static real_t Stepify(real_t s, real_t step)
+        public static real_t Snapped(real_t s, real_t step)
```

Comment:
The change in this file is good, except that you forgot to change Vector2.cs and Vector3.cs which use this method.

Reply:
Done.